contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/json/MaprDBJsonRecordReader.java

```diff
@@ -530,8 +530,10 @@ public void close() {
 
   @Override
   public String toString() {
-    StringBuilder sb = new StringBuilder("MaprDBJsonRecordReader[Table=")
-        .append(table.getPath());
+    StringBuilder sb = new StringBuilder("MaprDBJsonRecordReader[Table=");
+    if (table != null) {
+      sb.append(table.getPath());
+    }
     if (reader != null) {
       sb.append(", Document ID=")
```

Comment:
Also, please move `, `string here and below to the previous appenders.

Reply:
Made previous field to be present in any case. It seems there is no need in doing so.